examples/queue/amqp/amqp-sdk.js

```diff
@@ -0,0 +1,44 @@
+const { Reshuffle } = require('reshuffle')
+const { AMQPConnector } = require('reshuffle-amqp-connector')
+
+// This example uses reshuffle-amqp-connector
+// Code and documentation available on Github: https://github.com/reshufflehq/reshuffle-amqp-connector
+// Examples can be found in https://www.npmjs.com/package/amqplib
+// and in https://www.rabbitmq.com/tutorials/tutorial-two-javascript.html
```

Comment:
I think we normally treat 'examples' as Reshuffle examples. So in this case I suppose we are going to be referring to "More information" or something along these lines?